content/en/user-guide/aws/rds/index.md

```diff
@@ -1,83 +1,44 @@
+## Introduction
 
-## Supported DB engines
-
-Currently, it is possible to spin up PostgreSQL, MariaDB, MySQL, and MSSQL (SQL Server) databases on the local machine.
-
-{{< alert title="Note" >}}
-Some configuration of the RDS clusters and instances have currently only CRUD functionality. E.g., the `storage-encrypted` flag is returned as it is set, but there is no support for actual storage encryption yet. 
-{{< /alert >}}
-
-### Postgres Engine
-
-When creating an RDS DB cluster or instance with `postgres`/`aurora-postgresql` DB engine and a specific `EngineVersion`, LocalStack will install and provision the respective Postgres version on demand.
-
-Currently, major versions between 10 and 15 can be chosen - when selecting a major version outside of this range, the default version 11 is used as fallback.
-
-Please be aware that the minor version cannot be freely selected - the latest available version will be installed in the docker environment.
-
-In order to disable installation of custom versions, you may configure the environment variable `RDS_PG_CUSTOM_VERSIONS=0`, in which case always the default Postgres version 11 will be used.
-
-
-{{< alert title="Note" >}}
-The `describe-db-cluster` and `describe-db-instances` calls will still return the `engine-version` as it was defined for the creation, but the actual installed postgres engine could be different. This is important, e.g., when using a terraform configuration, as it should not detect changes in that case.
-{{< /alert >}}
-
-DB instances and DB cluster with Postgres engine support the creation and restoring of snapshots.
-
-### MariaDB Engine
-
-MariaDB will be installed as OS package in LocalStack. Currently, it is not possible to freely select a specific version. 
-
-Snapshots are currently not supported for MariaDB.
-
-### MySQL Engine
-
-By default, a MariaDB installation is used when requesting a MySQL engine type. 
-
-If you wish to use a real MySQL version, you can do so by setting the environment variable `RDS_MYSQL_DOCKER=1`. With this feature enabled, MySQL community server will be started in a new Docker container when requesting the MySQL engine. The `engine-version` will be used as the tag for the image, meaning you can freely select the desired MySQL version that is listed on the [official MySQL Docker Hub](https://hub.docker.com/_/mysql).
-
-In case you want to use a special image, you can also set the environment variable `MYSQL_IMAGE=<my-image:tag>`.
+RDS (Relational Database Service) is a managed database service provided by Amazon Web Services (AWS) that allows users to setup, operate, and scale relational databases in the cloud. RDS allows you to deploy and manage various relational database engines like MySQL, PostgreSQL, MariaDB, and Microsoft SQL Server. RDS handles routine database tasks such as provisioning, patching, backup, recovery, and scaling.
```

Comment:
not sure it's a good idea to outline this here, because LocalStack doesn't support this currently:
> backup, recovery, and scaling

Reply:
Its just for an intro to AWS RDS, not LocalStack's implementation :)